stories/DropdownLight/DropdownLight.regression-test.story.jsx

```diff
@@ -0,0 +1,22 @@
+import DropdownLight from '../../components/DropdownLight';
+import { itemsObjectMock, itemsStringMock } from './mock';
+
+export default {
+  title: 'DropdownLight',
+  component: DropdownLight,
+};
+
+const Template = args => <DropdownLight items={itemsObjectMock} {...args} />;
+
+const TemplateString = args => (
+  <DropdownLight items={itemsStringMock} {...args} />
+);
+
+export const Default = Template.bind({});
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  disabled: true,
+};
+
+export const WithString = TemplateString.bind({});
```

Comment:
The regressive tests aim to verify visual alterations. In this case, I don't see the need for this "WithString" test because it doesn't differ at all visually from the "Default" test
```suggestion
```

Reply:
I agree with you, I just created it to see if everything was working